Makefile

```diff
@@ -0,0 +1,35 @@
+DATE		= $(shell date)
+PYTHON		= $(shell which python)
+NOSETESTS = $(shell which nosetests)
```

Comment:
no preference, `master` was already using `nose` I just wanted to get it up and running.